### PR TITLE
Add Gecko XBL extension commands for anonymous nodes

### PR DESCRIPTION
**Pull request by andreastt**
See the commit messages on each of the patches.

### CODE REVIEW

File: py/selenium/webdriver/firefox/webdriver.py [hunk 1]
```diff
@@ -30,20 +30,40 @@
...
 from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
 from selenium.webdriver.firefox.extension_connection import ExtensionConnection
 from selenium.webdriver.firefox.firefox_profile import FirefoxProfile
 from selenium.webdriver.remote.webdriver import WebDriver as RemoteWebDriver
+from selenium.webdriver.remote.webelement import WebElement as RemoteWebElement
 from .service import Service
-from .options import Options
+
+
+class FirefoxWebElement(RemoteWebElement):
```
Comment (lukeis) [inline py/selenium/webdriver/firefox/webdriver.py]: i think this should get put into its own file. This could start to grow :)

**Comment by lukeis** [comment]
the py docs added here don't make it clear when one would use this... maybe a link to some mozilla documentation around what xbl is? or examples?

**Comment by andreastt** [comment]
@lukeis I addessed the documentation issue in f5445ed, but I really don’t know more about this than you do.

File: py/selenium/webdriver/firefox/webdriver.py (hunk 1 above)
Comment (andreastt) [inline py/selenium/webdriver/firefox/webdriver.py]: Fixed in f5445ed.

**Comment by lukeis** [comment]
Thanks :+1:

**Pull request merged**